src-lts/scripts/scr_catspeak_codegen/scr_catspeak_codegen.gml

```diff
@@ -1367,6 +1394,27 @@ function __catspeak_expr_while__() {
     }
 }
 
+/// @ignore
+///@return {Any}
+function __catspeak_expr_match__() {
+    var value_ = value();
+    
+    var i = 0;
+    var len = array_length(arms);
+    
+    repeat len {
+        var arm = arms[i];
+        
+        if arm.condition == undefined || value_ == arm.condition() {
```

Comment:
In the future, please put parens around `if` statements, like `if (condition)` instead of `if condition`.